### DISCUSSION TITLE
Navigator Seite (Links)

### DISCUSSION THREAD

**Discussion by Mephisto090** [mcuiobroker/docuJarvis#2] [Ideas]
Ich finde auch deine Leiste links könnte besser sortiert sein. Ich suche mich da echt ein Wolf und finde selten was. Wäre es nicht besser anderes zu sortieren. Wie findest du die Idee?

Beispiel:

-- Module
--- JsonTable
---- v2
----- TR064 - Nur aktive Geräte
----- .....
---- v3
----- JsonTable erstellen
----- ....

--- HTML Table
---- v2
----- .....
---- v3
----- ....

**Comment by mcuiobroker** [comment]
Das ist schwierig. Nicht von der Umsetzung, sondern eher von der Thematik, denn die v2 User suchen alles unter v2 und die anderen halt unter v3. 
Nun liegt die Chaotik auch ein wenig daran, dass die Doku keine echte Doku von einem fertigen Programm ist, sondern sich permanent ändert. Denn in einer Version von dem Programm kann schon wieder alles anders sein als vorher.
Dazu entwickelt sich das Programm zu schnell. Es gibt keinen stand der dann 100% funktioniert und auf dem man dann eine Dokumentation aufbauen könnte.
Dazu muss noch weiter entwickelt werden, aber man ist auf einem guten Weg.
Das Wiki / Doku (v2/v3) ist mehr alles nach meinen Erfahrungen und dem Wissendurst mit dem Programm und den Nachfragen von den anderen Usern entstanden.
Also ich bin genauso ein User, wie alle anderen, und habe keinen anderen Wissensstand bzgl. des Programmes als andere auch.
Ich mache die "Doku" aus Interesse und damit Zefau entlastet wird und sich somit auf die Programmierung konzentrieren kann.
Denn da ist noch einiges zu tun.

**Reply by Mephisto090** [reply]
Ich kann ja versuchen ein bisschen zu helfen, wo ich kann. Das Problem ist, dass es kein Changelog gibt von jeder Version Jarvis, um das Wiki anzupassen.

**Reply by mcuiobroker** [reply]
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/besonderheiten-v3/allgemein/system-neuigkeiten

https://mcuiobroker.gitbook.io/jarvis-infos/aenderungsprotokoll

**Reply by Mephisto090** [reply]
Das ist ja von dir. Du arbeitest ja echt viel ein und verfolgst das Repo. Respekt! :)

**Reply by mcuiobroker** [reply]
Die "Doku" hat mittlerweile ca. 700 Seiten. Da ist noch einiges zu tun?

**Reply by Mephisto090** [reply]
> Nicht von der Umsetzung, sondern eher von der Thematik, denn die v2 User suchen alles unter v2 und die anderen halt unter v3.

Wäre es nicht besser zwei Wikis zu haben? Eins für Version 2 und Version 3. So wird alles etwas übersichtlicher.

**Reply by mcuiobroker** [reply]
Es gibt für v2 eine eigene von Zefau. In meiner liegt die Hauptarbeit eher auf dem Bereich Styles / CSS Fokus. Dort ist sehr viel von den Usern gewünscht worden. Und das wurde nach jeder Anforderung dann dort mit aufgeführt.
Da v2 auch ein Auslaufmodell sein sollte, wird dort auch nicht mehr weitergearbeitet.

**Reply by Mephisto090** [reply]
Hast du Discord?

**Reply by mcuiobroker** [reply]
Nein. Nutze nur schriftlichen Kontakt.

**Reply by mcuiobroker** [reply]
Hast du unter ioBroker auch den Usernamen Mephisto090?
Frage nur weil ich es angebe in der Doku.

**Reply by Mephisto090** [reply]
> Nein. Nutze nur schriftlichen Kontakt.

Über Discord lässt sich ja auch Chatten nicht nur reden :)




> Hast du unter ioBroker auch den Usernamen Mephisto090?
> Frage nur weil ich es angebe in der Doku.

Meinst du im Forum?

**Reply by mcuiobroker** [reply]
Ja im Forum.

**Reply by Mephisto090** [reply]
jap `Mephisto090`

**Reply by Mephisto090** [reply]
Ich überlege mir mal wie man das ggf. noch etwas "aufräumen" kann

**Reply by mcuiobroker** [reply]
Jo, lass dir ein wenig Zeit.
